datasets/xk/companies/crawler.py

```diff
@@ -0,0 +1,291 @@
+        string = string.replace("$", "usd")
+        string = string.replace("%", "percent")
+        string = slugify(string, sep="_")
+    return string
+
+
+def norm_capital(context: Context, string: str) -> Dict[str, str]:
+    """
+    Normalize a capital string.
+    Args:
+        context: Context
+        string: str
+    Returns:
+        Capital as a dictionary of value and currency.
+    """
+    capital = {"value": "", "currency": ""}
+    try:
+        match = re.search(r"(?P<val>[0-9\.,]*)\s*(?P<cur>[^\d]+)?", string)
+        if match:
+            capital["value"] = match.groupdict().get("val")
+            capital["currency"] = norm_h(match.groupdict().get("cur"))
+    except Exception as exc:
+        context.log.warning(f"[Unable to parse capital: {exc}]")
+    return capital
+
+
+def parse_date(text: str) -> Any:
+    """
+    Parse a date from a string.
+    """
+    return h.parse_date(
+        text,
+        [
+            # 4/29/2014 12:00:00 AM
+            "%m/%d/%Y %I:%M:%S %p",
+            "%m/%d/%Y %H:%M:%S",
+        ],
+    )
+
+
+def get_the_key() -> str:
+    # Key and IV must be bytes, here assuming they are the same.
+    key = iv = API_KEY.encode("utf-8")  # Key and IV as bytes
+
+    # Message to encrypt
+    # Looks like we can safely use timestamp in the future.
+    data = (datetime.now(timezone.utc) + timedelta(seconds=60)).isoformat()
+
+    # Padding the data according to PKCS7
+    padder = PKCS7(algorithms.AES.block_size).padder()
+    padded_data = padder.update(data.encode()) + padder.finalize()
+
+    # Setting up the cipher configuration
+    cipher = Cipher(algorithms.AES(key), modes.CBC(iv), backend=default_backend())
+    # Encrypting the data
+    encryptor = cipher.encryptor()
+    encrypted = encryptor.update(padded_data) + encryptor.finalize()
+
+    return b64encode(encrypted).decode()
+
+
+def fetch_company(context: Context, company_id: int) -> None:
+    """
+    Fetch a single company from the Kosovo Registry of Business Organizations and Trade Names.
+    """
+    try:
+        resp = context.fetch_json(
+            "https://arbk.rks-gov.net/api/api/Services/TeDhenatBiznesit",
+            params={"nRegjistriId": company_id, "Gjuha": "en"},
+            headers={"key": get_the_key()},
+        )
+
+        if len(resp) != 1 or "teDhenatBiznesit" not in resp[0]:
+            context.log.error(f"Something strange with the {company_id})")
+
+        company = resp[0]["teDhenatBiznesit"]
+        activities = resp[0].get("aktivitetet", [])
+        founders = resp[0].get("pronaret", [])
+        representatives = resp[0].get("perfaqesuesit", [])
+
+        company_type = company.get("LlojiBiznesit")
+        if company_type not in COMPANY_TYPES:
+            context.log.warning("Unknown company type: ", company_type)
+
+        status = company.get("StatusiARBK")
+        if status not in STATUSES:
+            context.log.warning("Unknown status: ", status)
```

Comment:
I'm seeing unknown status warnings, but I think there's no need to put maintenance effort into these for now. I'd say just stick the Albanian in there.

Reply:
Let's use the albanian